docs/library/esp32.rst

```diff
@@ -88,6 +112,10 @@ Constants
 
     Used in `Partition.find` to specify the partition type.
 
+.. data:: MEM_DATA
+          MEM_EXEC
```

Comment:
Maybe call these constants `HEAP_DATA` and `HEAP_EXEC` to indicate they are related to the `idf_heap_info()` function, the key word being "heap"?